sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp

```diff
@@ -54,41 +53,45 @@ namespace Azure { namespace Data { namespace Tables {
      * @brief Include this parameter to specify that the tables' metadata be returned as part of
      * the response body.
      */
-    enum class ListTablesIncludeFlags
+    enum class QueryTablesIncludeFlags
     {
       None = 0,
       Metadata = 1,
     };
-    inline ListTablesIncludeFlags operator|(ListTablesIncludeFlags lhs, ListTablesIncludeFlags rhs)
+    inline QueryTablesIncludeFlags operator|(
+        QueryTablesIncludeFlags lhs,
+        QueryTablesIncludeFlags rhs)
     {
-      using type = std::underlying_type_t<ListTablesIncludeFlags>;
-      return static_cast<ListTablesIncludeFlags>(static_cast<type>(lhs) | static_cast<type>(rhs));
+      using type = std::underlying_type_t<QueryTablesIncludeFlags>;
+      return static_cast<QueryTablesIncludeFlags>(static_cast<type>(lhs) | static_cast<type>(rhs));
     }
-    inline ListTablesIncludeFlags& operator|=(
-        ListTablesIncludeFlags& lhs,
-        ListTablesIncludeFlags rhs)
+    inline QueryTablesIncludeFlags& operator|=(
+        QueryTablesIncludeFlags& lhs,
```

Comment:
Using a reference on POD (Plain Old Data) is often a mistake because it significantly hurts performance.

Does the `lhs` parameter really need to be passed by reference?

Reply:
done